### PR TITLE
updated ui tests and deleted unused ones

### PR DESCRIPTION
**Pull request by Velin92**

### CODE REVIEW

File: UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.reportContent-1.png
Comment (pixlwave) [inline UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.reportContent-1.png:1]: Is this correct, no snapshot any more?

File: UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.analyticsSettingsScreen.png
Comment (pixlwave) [inline UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.analyticsSettingsScreen.png:1]: It looks like this screen isn't reading the app name from the bundle correctly as it should be `Element X`

File: UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.reportContent-1.png
Comment (Velin92) [reply UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.reportContent-1.png:1]: yeah we have only 1 test for this screen now

File: UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.analyticsSettingsScreen.png
Comment (Velin92) [reply UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.analyticsSettingsScreen.png:1]: Uhm I think the name was just removed from the string no?

File: UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.roomLayoutBottom-1.png
Comment (pixlwave) [inline UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.roomLayoutBottom-1.png:1]: Keyboard is missing from this one

File: UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.analyticsSettingsScreen.png
Comment (Velin92) [reply UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.analyticsSettingsScreen.png:1]: yeah has been removed:
```
/// Share anonymous usage data to help us identify issues.
  public static var screenAnalyticsPromptHelpUsImprove: String { return L10n.tr("Localizable", "screen_analytics_prompt_help_us_improve") }
```

File: UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.serverSelection-1.png
Comment (pixlwave) [inline UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.serverSelection-1.png:1]: Keyboard

File: UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.roomLayoutBottom-1.png
Comment (Velin92) [reply UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.roomLayoutBottom-1.png:1]: done

File: UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.serverSelection-1.png
Comment (Velin92) [reply UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.serverSelection-1.png:1]: fixed

File: UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.userSessionScreen-3.png
Comment (pixlwave) [inline UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.userSessionScreen-3.png:1]: I guess the extra padding added to this view should only be when presented as a sheet and not as a popover.

Comment (Velin92) [reply UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.userSessionScreen-3.png:1]: fixed

File: ElementX/Sources/Screens/RoomScreen/View/RoomAttachmentPicker.swift [hunk 1]
```diff
@@ -18,6 +18,7 @@ import SwiftUI
 
 struct RoomAttachmentPicker: View {
     @ObservedObject var context: RoomScreenViewModel.Context
+    @Environment(\.isPresented) var isPresented
```
Comment (pixlwave) [inline ElementX/Sources/Screens/RoomScreen/View/RoomAttachmentPicker.swift:21]: Oh interesting it works even for the button itself. 😎

File: UITests/Sources/__Snapshots__/Application/pseudo-iPhone-14.bugReportWithScreenshot.png
Comment (pixlwave) [inline UITests/Sources/__Snapshots__/Application/pseudo-iPhone-14.bugReportWithScreenshot.png:1]: Something weird is going on your line spacing on the iPhone pseudo language screenshots. Look at how the English iPhone one doesn't change in layout, only colour, but the content in this one is taller for some reason 🤔

File: UITests/Sources/__Snapshots__/Application/pseudo-iPhone-14.roomPlainNoAvatar.png
Comment (pixlwave) [inline UITests/Sources/__Snapshots__/Application/pseudo-iPhone-14.roomPlainNoAvatar.png:1]: And its the same on all these timeline ones too, the timeline is taller for some reason 😕

**Review by pixlwave — APPROVED** [review]
LGTM other than the layout of iPhone-pseudo snapshots.

**Comment by Velin92** [comment]
> LGTM other than the layout of iPhone-pseudo snapshots.

I'll check what's wrong with that specific configuration on my iPhone, in the meantime, since you need to add more tests, can you generate the pseudo ones for iPhone 14 yourself?

**Pull request merged**